plugins/modules/nsxt_transport_node_profiles.py

```diff
@@ -254,6 +249,12 @@ def check_for_update(module, manager_url, mgr_username, mgr_password, validate_c
         transport_node_profile_with_ids.__contains__('host_switch_spec') and transport_node_profile_with_ids['host_switch_spec'].__contains__('host_switches') and \
         existing_transport_node_profile['host_switch_spec']['host_switches'] != transport_node_profile_with_ids['host_switch_spec']['host_switches']:
         return True
```

Comment:
Bugs Fixed - 2643929 - "tags" field is not support by "nsxt_transport_node_profiles" module